shared/src/commonMain/kotlin/com/boostyboys/mcs/data/api/models/match/LocalDateTimeSerializer.kt

```diff
@@ -0,0 +1,26 @@
+package com.boostyboys.mcs.data.api.models.match
+
+import kotlinx.datetime.LocalDateTime
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.toInstant
+import kotlinx.datetime.toLocalDateTime
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+
+object LocalDateTimeSerializer : KSerializer<LocalDateTime> {
```

Comment:
does this need null handling since it's being used on a nullable field?

Reply:
nah, kotlin serializer ignores null values by default - this is how UI renders a null date time
<img width="980" alt="Screenshot 2023-10-24 at 3 01 38 PM" src="https://github.com/Boosty-Boys/MidwestChampionshipSeriesComposeMultiplatformApp/assets/7951665/d2eb9dc7-db07-4bbc-904b-9e28e523be71">